benchbuild/experiment.py

```diff
@@ -246,6 +233,11 @@ def __add__(self, rhs):
         self.config.update(rhs.config)
 
 
-def discovered() -> tp.Dict[str, tp.Type[Experiment]]:
+ExperimentRegistry = tp.Dict[str, tp.Type[Experiment]]
+
+__REGISTRY__: ExperimentRegistry = {}
```

Comment:
Is a module level var here a good choice? In our case, we just put the registry as a class level variable, which encapsulates everything in one type.